packages/example/pages/_app.tsx

```diff
@@ -246,9 +254,36 @@ function RainbowKitApp({
 
   const [isMounted, setIsMounted] = useState(false);
   useEffect(() => setIsMounted(true), []);
+  // biome-ignore lint/nursery/useExhaustiveDependencies: TODO
+  useEffect(() => {
+    wagmiConfig.setConnectors(connectors);
+  }, [locale]);
 
   const appContextProps: AppContextProps = { authEnabled };
 
+  const locales = router.locales!;
+
+  const getUserDetectedBrowserLocale = () => {
+    // Start off with en_US locale
+    let locale = Locale.EN_US;
+
+    const detectedBrowserLocale = getDetectedBrowserLocale();
+
+    // If browser locale detected then see if we support the browser's locale
+    if (detectedBrowserLocale) {
+      // Defaults to en_US locale if no locale from browser found
+      locale = getSupportedLocaleFromBrowser(detectedBrowserLocale);
+    }
+
+    return locale;
+  };
+
+  // biome-ignore lint/nursery/useExhaustiveDependencies: TODO
+  useEffect(() => {
+    const userBrowserDetectedLocale = getUserDetectedBrowserLocale();
+    setLocale(userBrowserDetectedLocale);
+  }, []);
```

Comment:
Some `biome-ignore` for now. Will fix this later marked as `TODO`.

Here we will be detecting users `locale` from their browser using `navigator` 🙌